components/lua/luaconf.h

```diff
@@ -331,7 +331,7 @@ extern int readline4lua(const char *prompt, char *buffer, int length);
 ** avoids including 'stdio.h' everywhere.)
 */
 #if !defined(LUA_USE_STDIO)
-#define luai_writestring(s, l)  printf(s)
+#define luai_writestring(s, l)  fputs(s, stdout)
```

Comment:
It would probably be a good idea, since with `fwrite()` we can support printing binary strings (which someone might want to do if they're hooking up the UART to some peripheral).